pkg/dal/storage-client/data_access/mysql.go

```diff
@@ -460,7 +460,7 @@ func (s *StorageSqlImpl) QuerySloTraceDataWithPodUID(data interface{}, podUid st
 func (s *StorageSqlImpl) QueryCreateSloWithResult(data interface{}, requestParams *model.SloOptions) error {
 
 	if requestParams == nil || requestParams.Result == "" {
-		return fmt.Errorf("the params is error")
+		return fmt.Errorf("the params is error, delieryresult needed")
```

Comment:
Is it better to define the error code as a constant?

Reply:
I provide a custom error interface. we can classify errors by category, and define them globally, and return them uniformly through the interface.

Reply:
if we need to add a new error，such as NoPodUID，we just need to add it as shown below
![test](https://github.com/alipay/container-observability-service/assets/91560756/3f0b28df-b860-4fea-ac09-d08768a5bd16)